alpine/FieldContainer.hpp

```diff
@@ -29,30 +29,29 @@ class FieldContainer{
     std::shared_ptr<FieldLayout_t<Dim>> fl_m;
 
 public:
-    VField_t<T, Dim>& getE() { return E_m; }
-    void setE(VField_t<T, Dim>& E) { E_m = E; }
+    inline VField_t<T, Dim>& getE() { return E_m; }
+    inline void setE(VField_t<T, Dim>& E) { E_m = E; }
```

Comment:
Note that all these inlines are superfluous. All functions entirely defined in the class are already inlined. Read for example https://en.cppreference.com/w/cpp/language/inline

Reply:
@aaadelmann shall I remove `inline` from all the get/set functions?  As @matt-frey pointed out, it seems to be redundant.

Reply:
I removed `inline` everywhere. See dedf58a